File: test/fixtures/php-guzzle/j_data_priority_than_f.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->post('http://localhost:28139/post', [
    'headers' => [
        'Content-Type' => 'application/x-www-form-urlencoded'
    ],
    'form_params' => [
        'data1' => 'data1',
        'data2' => 'data2',
        'data3' => 'data3'
    ]
]);

File: test/fixtures/java-unirest/j_data_priority_than_f.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.post("http://localhost:28139/post")
            .header("Content-Type", "application/x-www-form-urlencoded")
            .field("data1","data1")
            .field("data2","data2")
            .field("data3","data3")
            .asString();
    }
}
